src/fsharp/FSharp.Build/xlf/FSBuild.txt.cs.xlf

```diff
@@ -2,6 +2,11 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="cs" original="../FSBuild.resx">
     <body>
+      <trans-unit id="fSharpBannerVersion">
+        <source>{0} for F# {1}</source>
+        <target state="new">{0} for F# {1}</target>
+        <note />
```

Comment:
Might want to change this to: `<note>Contains version numbers.  E.g., '10.4.3 for F# 4.6'</note>` so the translators know what's happening.

Reply:
I like that … good idea.

Reply:
I tried it, and the note I had added got overwritten.